src/payment/data/constants.js

```diff
@@ -15,6 +52,9 @@ export const CERTIFICATE_TYPES = {
  * @see POLLING_PAYMENT_STATES
  *
  * @note: This enum is unique to Commerce Coordinator backend.
+ *
+ * @readonly
+ * @enum {string|Symbol}
  */
 export const PAYMENT_STATE = (((webserviceEnum = {
```

Comment:
Nit: Can we replace `// Our Additions` on line 80 below with an explanation that the payment states listed below are added by Payment MFE to the backend's list of possible payment states?